test/XRP/fakes/fake-xrp-transaction-type-protobufs.ts

```diff
@@ -62,4 +72,26 @@ testAccountSetProtoAllFields.setTickSize(testTickSizeProto)
 const testAccountSetProtoOneFieldSet = new AccountSet()
 testAccountSetProtoOneFieldSet.setClearFlag(testClearFlagProto)
 
-export { testAccountSetProtoAllFields, testAccountSetProtoOneFieldSet }
+// AccountDelete protos
+const testAccountAddressProto = new AccountAddress()
```

Comment:
same here - not sure which it's supposed to be

Reply:
good nit - this actually gets taken care of in a downstream PR I believe so I can be negligent here